vars/piperPipelineStageInit.groovy

```diff
@@ -106,7 +115,10 @@ void call(Map parameters = [:]) {
     def stageName = StageNameProvider.instance.getStageName(script, parameters, this)
 
     piperStageWrapper (script: script, stageName: stageName, stashContent: [], ordinal: 1, telemetryDisabled: true) {
-        def scmInfo = checkout(parameters.checkoutMap ?: scm)
+        def scmInfo = parameters.scmInfo
+        if (!parameters.skipCheckout) {
```

Comment:
[clarification] In case `parameters.skipCheckout` is not present at all the checkout happens. If it is `false` the checkout happens, too. In case it is `true` the checkout is skipped. :white_check_mark:

Reply:
Reading this maybe it would be better to make it a proper boolean, because if it is the String `"false"` it will also **not** checkout! What are your thoughts?

Reply:
Yes, string `"false"` might happen easily in case someone provides a string (->"false") rather than a boolean in some config file.. You are right, we should avoid that.

Reply:
So we throw an error if it's not a boolean or null, I would suggest.

Reply:
Done